pkg/synopsysctl/cmd_destroy.go

```diff
@@ -197,7 +210,9 @@ func destroy(namespace string) error {
 		}
 	}
 
-	log.Infof("successfully submitted destroy Synopsys Operator in namespace '%s'", namespace)
+	if isExist {
```

Comment:
This isn't a reliable state of the result of the action.  It's used and overwritten multiple times and since logs are printed instead of errors returned, this statement can be reached after errors are thrown.  Suggest re-working logic to return errors instead of printing logs, then print log if deletion is successful.  Probably better to to move this success message where the NS is actually deleted too.

Reply:
done